tests/braille-specs/sk.yaml

```diff
@@ -0,0 +1,70 @@
+﻿# Tests for Slovak braille
+#
+#  Copyright (C) 2021-2022 the Braille Authority of Slovakia <https://www.skn.sk/en/slovak-authority-for-the-braille-code>
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
```

Comment:
Could we perhaps change this into something more permissive? Some other test files have this:

>     # Copying and distribution of this file, with or without modification,
>     # are permitted in any medium without royalty provided the copyright
>     # notice and this notice are preserved. This file is offered as-is,
>     # without any warranty.

Reply:
Thank you for the hint. Fixed.

Reply:
@pvagner You should have kept the copyright lines though. It helps identifying who added certain tests and who should be contacted if there are questions regarding those tests.

Reply:
By the way, the file was originally added by Simon Aittamaa from Index in 2017, and was originally supplied by Mr. Mamojka. So it should say:

    Copyright (C) 2017,2021-2022 the Braille Authority of Slovakia <https://www.skn.sk/en/slovak-authority-for-the-braille-code>